resources/uk/gov/hmcts/helm/warn-about-workload-identity.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+
+CHART_DIRECTORY=${1}-${2}
+cd charts/"${CHART_DIRECTORY}" || exit 10
+# Check if app still has aad config
+grep --quiet  "aadIdentityName:" values.yaml
```

Comment:
Better to check in values.*.yaml ? like preview and aat?

Reply:
Yup good idea, I added to search all .yaml files in general in case they're named differently